website/src/content/help-needed.mdx

```diff
@@ -49,82 +49,7 @@ If you are interested in more advanced functionalities, let us know and open a [
-
-Let's say there are two deployments of the same application: `A` and `B`. We can assign an environment value for the `entryPointUriPath` property in its configuration.
-
-```json
-{
-  "entryPointUriPath": "${env:ENTRY_POINT_URI_PATH}"
-}
-```
-
-The `ENTRY_POINT_URI_PATH` environment variable can be provided in different ways.
-
-For example:
-- as an inline environment variable when running a script command.
-
-  ```console
-  ENTRY_POINT_URI_PATH=my-application-entry-point mc-scripts start
-  ```
-
-- using a [dotenv](https://github.com/motdotla/dotenv) file.
-
-  ```bash title=".env-A"
-  ENTRY_POINT_URI_PATH=my-application-entry-point
-  ```
-
-  ```console
-  mc-scripts --env .env-A start
-  ```
-
-- by defining the environment variables in your CI service.
-
-Using this pattern, different `entryPointUriPath` values can be defined for different environments:
-* `A` environment: `my-application-entry-point`
-* `B` environment: `my-application-entry-point-test`
-
-With this setup, preview versions of the Custom Application can be deployed to the `B` environment and tested using the application `my-application-entry-point-test` entry point path.
-
-However, besides using different `entryPointUriPath` values, the logic of verifying the user permissions in the Custom Application source code must also be adjusted accordingly.
-
-As [explained in this section](/migrating-from-project-level-custom-applications#choose-the-entrypointuripath), the permission names of a Custom Application are derived from the `entryPointUriPath` value. Therefore, when using different `entryPointUriPath` values, the permission names must also be changed.
-
-We recommend using the helper function `entryPointUriPathToPermissionKeys` to compute the permission names from the `entryPointUriPath`.
-
-Find more information on how to apply user permissions in the [Permissions](/development/permissions) documentation.
-
-<Info>
-
-In this scenario the `entryPointUriPath` value cannot be defined statically but it's injected using an environment variable.
-
-Therefore, both server-side and client-side logic must be considered when reading the value from the source code.
-
-```js highlightLines="4-6"
-// Make sure to import the helper functions from the `ssr` entry point.
-import { entryPointUriPathToPermissionKeys } from '@commercetools-frontend/application-shell/ssr';
-
-export const entryPointUriPath = typeof window === 'undefined'
-  ? process.env.ENTRY_POINT_URI_PATH
-  : window.app.entryPointUriPath;
-
-export const PERMISSIONS = entryPointUriPathToPermissionKeys(entryPointUriPath);
-```
-</Info>
+You can set up a test or staging environment for your Custom Application by adding a [deployment preview](./api-reference/deployment-previews).
```

Comment:
Nit: Should we maybe leave the old docs and add an info message explaining there is now a better way of doing this?
At least for the time being, then we can remove the old content.

Reply:
Keeping the old content will be contradictory. But I'd like to understand the "At least for the time being" part to make a better point.

Reply:
@emmenko while changing this document, I reliazied that we should keep the older docs for teams that are using the workaround. Hence, I've restructured the section to keep both but labeling old approach as "workaround" and mentioning deployment previews at top.